src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    Optional<MovieSession> get(Long id);
```

Comment:
```suggestion
    MovieSession get(Long id);
```

Dao should return Optional where we can get null, but services should return objects

Reply:
)) I see that I need to review lesson about Optional...